src/views/modules/bar.php

```diff
@@ -60,14 +65,21 @@ class="tribe-bar-views-select tribe-no-param tribe-accessible-js-hidden"
 		<?php if ( ! empty( $filters ) ) { ?>
 			<div class="tribe-bar-filters">
 				<div class="tribe-bar-filters-inner tribe-clearfix">
+					<h3 class="tribe-events-visuallyhidden"><?php printf( esc_html__( '%s Search', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h3>
```

Comment:
Could we use `tribe-events-visually-hidden`

Reply:
@bordoni (same question as above)

Reply:
Ohh sorry, didn't realize that. Let's keep it as is.